ironfish/src/consensus/verifier.ts

```diff
@@ -66,10 +66,14 @@ export class Verifier {
       const fee = transactionFees[i]
 
       // Miner's fee should be only the first transaction
-      if ((i === 0 && fee > 0) || (i !== 0 && fee < 0)) {
+      if (i === 0 && fee > 0) {
         return { valid: false, reason: VerificationResultReason.INVALID_MINERS_FEE }
       }
 
+      if (i !== 0 && fee < 0) {
+        return { valid: false, reason: VerificationResultReason.INVALID_TRANSACTION_FEE }
+      }
```

Comment:
Nice change here, this is a good improvement 👍

Reply:
Thank you!